sycl/doc/extensions/experimental/sycl_ext_oneapi_bfloat16_math_functions.asciidoc

```diff
@@ -186,6 +186,194 @@ T fabs(T x);
 
```

Comment:
We need to decide whether this extension is versioned.  Experimental extensions are allowed to be versioned, but they do not need to be.  Here are the options:

* We decide that this extension is versioned.  In this case, bump the version number of the feature-test macro to `2`, with a note indicating that these functions were added in that versions.  In addition, add a sentence to each description that says "This function is available starting in version 2 of this extension."

* We decide that this extension is not versioned.  In this case, change the text for the feature-test macro to say "The APIs of this experimental extension are not versioned, so the feature-test macro always has this value" as described in the [template](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/template.asciidoc#feature-test-macro).

@JackAKirk, do you have an opinion on which to do?

Reply:
I would guess that it is OK to not version it. I think it would make sense to version it if any of the existing APIs are likely to change, but for this extension this doesn't seem very likely.

Reply:
Since I seed that you are updating this PR, I just wanted to point out that this comment still needs to be addressed.  Removing the versioning is OK (since this is an experimental extension).  We should do one of the two things I list above, though.

Reply:
Hi, @gmlueck 
I updated the feature test macro as not versioned, could you help review again?
Thanks very much.